gapis/api/gles/compat.go

```diff
@@ -562,6 +562,13 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 				log.E(ctx, "Error decompressing texture: %v", err)
 			}
 
+		case *GlTexBufferEXT:
+			if !version.IsES && version.AtLeast(3, 1) { // Strip suffix on desktop.
```

Comment:
Of course it does, in order to know how compat behaves, you need to know the exact version and list of all supported extensions on the target machine.
We might one day wonder why this does not work for someone until we think to ask if he is using 3.0 by any chance.